packages/api/src/external/fhir/document/index.ts

```diff
@@ -327,7 +327,7 @@ export function containedPatientToFHIRResource(
       resourceType: "Patient",
       id: chosenResourceId,
       address: convertCWAdressToFHIR(resource.address),
-      gender: convertCWGenderToFHIR(resource.gender?.coding),
+      gender: convertCWGenderToFHIR(resource.gender?.coding as unknown as Gender[]), // TODO ENG-200 - Remove the casting
```

Comment:
had to do these b/c was getting a compilation error I couldn't fix after a few attempts, it was becoming a rabbit hole.